### DISCUSSION TITLE
Graylog and Ingress challange for a newbie

### DISCUSSION THREAD

**Discussion by githubmame** [artifacthub/hub#3631] [Q&A]
Hi
I am a newbie on Linux, Kubernetes and Helm at all. Trying to follow some guides and how to to improve my knowledge.
Therefore please be patient with me.

Following is my case

**HOWTO:**
I did follow this tutorial ... yeah I am German speaking ;-)
https://tutorials-raspberrypi.de/supercomputer-raspberry-pi-kubernetes-cluster/

**HARDWARE:**
several RaspberryPI in a Kubernetes Cluster

**VERSION:**
helm version
BuildInfo{Version:"v3.14.0", GitCommit:"3fc9f4b2638e76f26739cd77c7017139be81d0ea", GitTreeState:"clean", GoVersion:"go1.21.5"}

kubectl version
Client Version: v1.28.5+k3s1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.28.5+k3s1

kubectl create namespace graylog

**COMMAND:**
helm install --namespace "graylog" graylog kongz/graylog

**OUTPUT:**
NAME: graylog
LAST DEPLOYED: Sun Jan 28 18:04:09 2024
NAMESPACE: graylog
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
To connect to your Graylog server:

1. Get the application URL by running these commands:

  Graylog Web Interface uses JavaScript to get detail of each node. The client JavaScript cannot communicate to node when service type is `ClusterIP`.
  If you want to access Graylog Web Interface, you need to enable Ingress.
    NOTE: Port Forward does not work with web interface.

2. The Graylog root users

  echo "User: admin"
  echo "Password: $(kubectl get secret --namespace graylog graylog -o "jsonpath={.data['graylog-password-secret']}" | base64 --decode)"

To send logs to graylog:

  NOTE: If `graylog.input` is empty, you cannot send logs from other services. Please make sure the value is not empty.
        See https://github.com/KongZ/charts/tree/main/charts/graylog#input for detail

**QUESTION:**
how I enable Ingress? I have no idea where and how to start at all. Sorry.
Therefore I am happy for any basic support.

Thanks mame

**Comment by tegioz** [comment]
Hi @githubmame 👋

For issues with the content listed on artifacthub.io it's best to contact the particular project directly. In the links section of the right panel in the package view you usually may find some links that can point you to the project's repository or the maintainers emails.

Hope this helps 🙂

**Reply by githubmame** [reply]
Hi @tegioz 
Thanks for your reply and I will try the direct contact via email.
THX mame
